packages/merge/src/typedefs-mergers/directives.ts

```diff
@@ -8,11 +8,35 @@ import {
 import { isSome } from '@graphql-tools/utils';
 import { Config } from './merge-typedefs.js';
 
+const isLinkDirective = (directive: DirectiveNode): boolean => directive.name.value === 'link';
```

Comment:
not sure if we already have AST narrowing fns. Further, I didn't see a `LinkDirective` type, which woulda been nice. Not going to add anything like that without discussion :)